README.md

```diff
@@ -38,15 +37,9 @@ To get a local copy up and running follow these simple example steps.
 
 
 
-## Authors
+## Author
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+👤 **Zain Sadaqat**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
```

Comment:
Kindly add social media links to your account.

Reply:
Great job. This comment is constructive and delivered in a great manner. But It doesn't explain why the suggested change is required, which will benefit the developer. Instead of this comment, you can rephrase it like `Kindly add social media links to your readme so that the person visiting your repo won't find it difficult to find who you are.`